src/Common/Core/Impl/Extensions/CharExtensions.cs

```diff
@@ -3,7 +3,7 @@
 
 namespace Microsoft.Common.Core {
     public static class CharExtensions {
-        public static readonly char[] LineBreakChars = new char[] { '\n', '\r' };
+        public static char[] LineBreakChars => new char[] { '\n', '\r' };
```

Comment:
`public static char[] LineBreakChars { get; } = new char[] { '\n', '\r' };`
Otherwise it will create new array every time you access the property